pubtools/exodus/hooks/pulp.py

```diff
@@ -0,0 +1,165 @@
+import logging
```

Comment:
Can this directory structure please be named in a way which shows it as being private?  Like naming the directory `_hooks`.  We don't want people to import and use this code directly.

You will also need to do the "entry points" thing documented at https://release-engineering.github.io/pubtools/hooks.html#guide-hook-implementers .  If you don't, then when a task starts there's no guarantee this ever gets imported (and so hooks might not be registered).